src/components/schedule-visualizer/tests/robots-overlay.test.tsx

```diff
@@ -7,20 +7,46 @@ import ColorManager from '../colors';
 import Robot from '../robot';
 import fakeFleets from '../../../mock/data/fleets';
 
-test('Render robots correctly', () => {
-  const robots = fakeFleets()[0].robots;
-  const bounds = new L.LatLngBounds([0, 25.7], [-14, 0]);
-  const colorManager = new ColorManager();
-  // TextEncoder is not available in node
-  colorManager.robotColor = jest.fn(async () => 'black');
-  colorManager.robotColorFromCache = jest.fn(() => 'black');
-  const wrapper = mount(
-    <LMap>
-      <RobotsOverlay bounds={bounds} colorManager={colorManager} robots={robots} />
-    </LMap>,
-  );
-  expect(wrapper.find(Robot).exists()).toBeTruthy();
-  expect(wrapper.find(Robot).length).toBe(robots.length);
+const robots = fakeFleets()[0].robots;
+const bounds = new L.LatLngBounds([0, 25.7], [-14, 0]);
+const colorManager = new ColorManager();
+// TextEncoder is not available in node
+colorManager.robotColor = jest.fn(async () => 'black');
+colorManager.robotColorFromCache = jest.fn(() => 'black');
+let conflictRobotNames: string[][] = [];
 
-  wrapper.unmount();
+describe('Robots Overlay', () => {
+  test('Render robots correctly', async () => {
+    const wrapper = mount(
+      <LMap>
+        <RobotsOverlay
+          bounds={bounds}
+          colorManager={colorManager}
+          robots={robots}
+          conflictRobotNames={conflictRobotNames}
+        />
+      </LMap>,
+    );
+
+    expect(wrapper.find(Robot).exists()).toBeTruthy();
+    expect(wrapper.find(Robot).length).toBe(robots.length);
+
+    wrapper.unmount();
+  });
+
+  test('Red shadow appears when robots are in trajectory conflict', async () => {
+    conflictRobotNames = [[fakeFleets()[0].robots[0].name]];
+    const wrapper = mount(
+      <LMap>
+        <RobotsOverlay
+          bounds={bounds}
+          colorManager={colorManager}
+          robots={robots}
+          conflictRobotNames={conflictRobotNames}
+        />
+      </LMap>,
+    );
+    expect(wrapper.containsMatchingElement(<feDropShadow floodColor={'red'} />)).toBeTruthy();
```

Comment:
If you use theme.palette.error.main maybe you would have a problem here?

Reply:
```
expect(wrapper.containsMatchingElement(<feDropShadow floodColor="#f44336" />)).toBeTruthy();
```
this is the simplest workaround that I can think of: to take the actual value of the red used in the theme object. Would it be bad practice?

Reply:
Mmm.. how about creating a new theme since we are using the default 
`const theme = createMuiTheme();`
With that, you will have access to `theme.palette.error.main`.
`floodColor={theme.palette.error.main}`
If in the future we want to create a custom theme, we would also be able to access that configuration in this test (if it's exported, of course).  

HTH,